lib/src/kbasesearchengine/events/exceptions/Retrier.java

```diff
@@ -110,6 +112,10 @@ public RetryLogger getLogger() {
                 } else {
                     retries++;
                 }
+            } catch (IOException | IndexingConflictException e) {
```

Comment:
The Retrier should definitely not retry non-retriable errors.

Reply:
ops. came from intellij auto correct. will revert